test/fixtures/experimental/decorators-stage-2/no-computed-decorator-member/actual.js

```diff
@@ -0,0 +1,6 @@
+class Foo {
+  @bar[bizz]
```

Comment:
This is a great test; you could also add tests for cases like `@(bar.baz)` and `@foo().bar`, which one might expect to work. The grammar is extremely restrictive here.